very easy/27.js

```diff
@@ -1,5 +1,5 @@
 function equalSlices(total, people, each) {
-	
+	return total/(people*each)>=1;

```

Comment:
Why in the conditional >=1 not >0 ? Are you sure that dividing by zero could be good solution?